packages/govern-subgraph/src/GovernQueue.ts

```diff
@@ -82,79 +81,64 @@ export function handleScheduled(event: ScheduledEvent): void {
 }
 
 export function handleExecuted(event: ExecutedEvent): void {
-  const container = loadOrCreateContainer(event.params.containerHash)
+  let container = loadOrCreateContainer(event.params.containerHash)
   container.state = EXECUTED_STATUS
   container.save()
 }
 
 export function handleChallenged(event: ChallengedEvent): void {
-  const queue = loadOrCreateQueue(event.address)
-  const container = loadOrCreateContainer(event.params.containerHash)
+  let queue = loadOrCreateQueue(event.address)
+  let container = loadOrCreateContainer(event.params.containerHash)
 
   container.state = CHALLENGED_STATUS
 
-  handleContainerEvent(
-    container,
-    event.block.timestamp,
-    CHALLENGE_CONTAINER_EVENT,
-    [
-      event.params.actor.toHex(),
-      event.params.reason.toHex(),
-      ConfigEntity.load(queue.config).resolver.toHex(),
-      event.params.resolverId.toHex(),
-      event.params.collateral.token.toHex(),
-      event.params.collateral.amount.toHex()
-    ]
-  )
+  let containerEvent = handleContainerEventChallenge(container, event)
+
+  // TODO
```

Comment:
Why we left this as a TODO?

Reply:
oops completely forgot this − I need to check something with you about it!